README.md

```diff
@@ -242,6 +246,19 @@ CREATE TABLE checks (name TEXT NOT NULL PRIMARY KEY, timestamp INT8, results TEX
 
 Currently the status page does not support SQL storage.
 
+#### Slack notifier
+
+slack configuration:
+```json
+{
+	"name": "slack",
+	"username": "username",
+	"channel": "#channel-name",
+	"webhook": "webhook-url"
+}
+```
+
+How to [create webhook](https://get.slack.help/hc/en-us/articles/115005265063-Incoming-WebHooks-for-Slack)
```

Comment:
Use a full sentence, like "Follow these instructions to create a webhook."

Reply:
done